cmake/KokkosConfig.cmake.in

```diff
@@ -84,8 +86,10 @@ function(kokkos_check)
       set(KOKKOS_CHECK_SUCCESS FALSE)
     endif()
   endforeach()
-  if(NOT KOKKOS_CHECK_SUCCESS)
+  if(NOT KOKKOS_CHECK_SUCCESS AND NOT KOKKOS_CHECK_RESULT)
     message(FATAL_ERROR "Kokkos does NOT provide all backends and/or architectures requested")
+  else()
+    set(${KOKKOS_CHECK_RESULT} ${KOKKOS_CHECK_SUCCESS} PARENT_SCOPE)
```

Comment:
@jjwilke You missed that one.  OK to merge after this is fixed.

Reply:
Correction. I missed *2*.